src/ert/run_models/iterated_ensemble_smoother.py

```diff
@@ -110,7 +110,7 @@ def analyzeStep(
             )
         except ErtAnalysisError as e:
             raise ErtRunError(
-                f"Analysis of simulation failed with the following error: {e}"
+                f"Analysis of experiment failed with the following error: {e}"
```

Comment:
I think we can be a bit more concrete on what failed here: 
```suggestion
                f"Update algorithm failed with the following error: {e}"
```